lib/coverband/collectors/view_tracker.rb

```diff
@@ -20,6 +20,7 @@ class ViewTracker
 
       def initialize(options = {})
         raise NotImplementedError, "View Tracker requires Rails 4 or greater" unless self.class.supported_version?
+        raise "Coverband: view tracker initialized before configuration!" if !Coverband.configured? && ENV["COVERBAND_TEST"] == "test"
```

Comment:
Why do we only want to raise this error for `ENV["COVERBAND_TEST"] == "test"`?

Reply:
I was worried about this raising in various scenarios like disabling coverband in RAILS env test or not having a redis in some CI build step, so I wanted to start with just something that let us test and verify the fix... I will keep looking if we can turn it on more extensively